### PR TITLE
OPEN-32, OPEN-43: Improve services GUI

### PR DESCRIPTION
**Pull request by delrueba**
OPEN-32 & OPEN-43

Main issues: updated table layout & provided 'deactivate' button.
Also moved search field and buttons to improve ux and match other pages.

### CODE REVIEW

File: resources/assets/js/page/PageHome.vue [hunk 1]
```diff
@@ -1,31 +1,36 @@
...
+          <button type="button" class="btn btn-default" @click="requestService">Vraag toegang tot een dienst</button>
+        </span>
+          <form class="pull-right">
+            <div class="form-group">
+                <input v-model="query" @input="route.offset=0" class="form-control" :placeholder="'Zoek ' + (route.tab ? 'gebruikers' : (draft ? 'inactive':'actieve') + ' diensten')" style="max-width:300px" type="search">
+            </div>
+        </form>
 
-      <button type="button" class="btn btn-success" @click="newUser" v-if="route.tab == 'users'">Gebruiker uitnodigen</button>
+      <!-- <button type="button" class="btn btn-link btn-disabled" :class="{active: route.tab=='admin'}" @click="route.tab='admin'" disabled>Toon administrators</button> -->
```
Comment (Jelle-S) [inline resources/assets/js/page/PageHome.vue]: Remove this comment.

File: resources/assets/js/page/PageHome.vue [hunk 2]
```diff
@@ -1,31 +1,36 @@
...
-    </div>
-    <div v-else>
-      <button type="button" class="btn btn-default" @click="requestService">Vraag toegang tot een dienst</button>
-    </div>
+      <!--<div class="btn-group" v-if="!route.tab">-->
+        <!--<button type="button" class="btn btn-default" :class="{ 'btn-success': !draft }" @click="draft = false">Toon actieve diensten</button>-->
+        <!--<button type="button" class="btn btn-default" :class="{ 'btn-warning': draft }" @click="draft = true">Activeer diensten</button>-->
+      <!--</div>-->
+
+      </div>
```
Comment (Jelle-S) [inline resources/assets/js/page/PageHome.vue]: Remove these comments.

File: resources/assets/sass/app.scss [hunk 3]
```diff
@@ -24,17 +24,26 @@ body {
 
 .container-breadcrumb {
   margin-bottom: 0;
-  border-bottom: 1px solid #ccc;
+  //border-bottom: 1px solid #ccc;
```
Comment (Jelle-S) [inline resources/assets/sass/app.scss]: Remove this comment.

File: resources/assets/sass/app.scss [hunk 4]
```diff
@@ -24,17 +24,26 @@ body {
...
 }
 .breadcrumb > .active > a {
   color: #777;
-  text-decoration: underline;
+  //text-decoration: underline;
```
Comment (Jelle-S) [inline resources/assets/sass/app.scss]: Remove this comment.

File: resources/assets/sass/app.scss [hunk 5]
```diff
@@ -55,15 +64,20 @@ body {
     padding: 5em 1em;
   }
 }
-.table > tbody + tbody {
-  border-top-width: 1px;
+//.table > tbody + tbody {
+//  border-top-width: 1px;
+//}
```
Comment (Jelle-S) [inline resources/assets/sass/app.scss]: Remove this comment.

File: resources/assets/sass/app.scss [hunk 6]
```diff
@@ -73,17 +87,41 @@ body {
 
 .table {
   margin-top: 15px;
-  border-top: 1px solid #999;
+  //border-top: 1px solid #999;
```
Comment (Jelle-S) [inline resources/assets/sass/app.scss]: Remove this comment.

File: resources/assets/sass/app.scss [hunk 7]
```diff
@@ -73,17 +87,41 @@ body {
...
+  margin-bottom: 80px;
+
+
+
+  >thead{
+    background-color: #DCE9ED;
+  }
   >thead>tr>th {
-    border-bottom: 1px solid #999;
+    //border-bottom: 1px solid #999;
```
Comment (Jelle-S) [inline resources/assets/sass/app.scss]: Remove this comment.

File: resources/assets/sass/app.scss [hunk 8]
```diff
@@ -232,7 +270,7 @@ h1+h3 {
   right: 0;
   bottom: 0;
   background: white;
-  border-top: 1px solid #ccc;
+  //border-top: 1px solid #ccc;
```
Comment (Jelle-S) [inline resources/assets/sass/app.scss]: Remove this comment.

File: resources/assets/sass/app.scss [hunk 9]
```diff
@@ -73,17 +87,41 @@ body {
 
 .table {
   margin-top: 15px;
-  border-top: 1px solid #999;
+  //border-top: 1px solid #999;
+  border-collapse:collapse;
+  margin-bottom: 80px;
+
+
```
Comment (Jelle-S) [inline resources/assets/sass/app.scss]: Only one newline needed.

File: resources/assets/sass/app.scss [hunk 10]
```diff
@@ -55,15 +64,20 @@ body {
...
+  background-color: #F8F8F8;
 }
 .text-danger > a {
   color: inherit;
 }
 .td-btn {
   padding-top: 4px !important;
   padding-bottom: 4px !important;
+  width: 1px;
+
```
Comment (Jelle-S) [inline resources/assets/sass/app.scss]: Remove newline before closing brace.

**Comment by delrueba** [comment]
## Style changes
Based on the mockups provided with the tickets. 

### General
- colors (~styleguide)
- search field: positioned near table
- pagination: match width of table
- extra indentation for sortable table cells
- updated breadcrumbs layout

### No active services
- changed button layout to match channels page (last image)

afbeelding

### Activate services
- replaced 'empty checkbox' with textual button

afbeelding

### Active services
- added 'deactivate' option

afbeelding

### Channels
- for comparison only
afbeelding

**Pull request merged**